var/spack/repos/builtin/packages/compadre/package.py

```diff
@@ -56,6 +58,10 @@ def cmake_args(self):
         if "+mpi" in spec:
             options.append("-DCompadre_USE_MPI:BOOL=ON")
 
+        if "~tests" in spec:
+            options.append("-DCompadre_EXAMPLES:BOOL=OFF")
+            options.append("-DCompadre_TESTS:BOOL=OFF")
```

Comment:
Just so I understand, these options default in the build to `ON`, is that correct?

Reply:
I ask because I usually see the opposite.  If the variant is enabled then the corresponding options are set to `True` or `ON`.

Reply:
These do default to `ON` in the build (they do now as well, prior to this PR). This PR allows for the tests/examples to be turned off.